backend/src/db/providers/clientCourseProvider.ts

```diff
@@ -22,9 +24,13 @@ const applyCourseProvider = async (courseId: string, userId: string, progressDto
   const applyedCourse = await ClientCourseModel.create({
     user: userId,
     course: courseId,
-    status: CourseStatus.approved,
+    status: CourseStatus.pending,
     progress: progressDto,
   });
+  await UserModel.updateMany(
+    { role: USER_ROLES.MANAGER },
+    { $push: { pendingCourses: applyedCourse } },
+  );
```

Comment:
i think we need to pass additional agr ManagerId to update only manager we need not all as in this case
cause here all manager will get this new applyed course

Reply:
done